WooCommerce/Classes/Tools/VersionHelpers.swift

```diff
@@ -0,0 +1,135 @@
+import Foundation
+
+/// Helpers for working with versions (e.g. comparing two version strings)
+///
+final class VersionHelpers {
```

Comment:
nit: since comparing WordPress plugin versions is quite different from other versions like app versions, it might be worth specifying that this is for WordPress plugin versions?

```suggestion
final class WordPressPluginVersionHelpers {
```

Reply:
> since comparing WordPress plugin versions is quite different from other versions like app versions

In what ways? The only difference I am aware of is that we tend to avoid double digit minors and roll majors to do so.

I'd like to keep this as generic as possible.